Alto-1654: Readme updates (#6)
altostra/altostra-template-static-website: README.md

# Static Website Template
Use this template to create a static website that is published over a CDN.

## Getting started

### Things you'll need 
1. AWS [SAM-CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
1. An Altostra Account( Don't have one yet? Just [login](https://app.altostra.com) here)
1. Altostra CLI Installed ([how?](../reference/CLI/altostra-cli.html#installation))
1. VSCode with Altostra Editor ([how?](../getting-started/installation.html#install-the-visual-studio-code-extension))
1. A Connected AWS Account ([how?](../getting-started/connect-your-accounts.html#connect-your-cloud-service-accounts))

> 1. An Environment connected to your AWS Account ([how?](../howto/envs/manage-environments.html)) - We'll call it `Dev` for berevity, but you can pick any of your environments

Reviewer: Update with latest from Yossi, he changed it a bit.

### Using the template

You have several options to get started with this template.  
Either go to the Altostra Web Console and create a new project.  

> When asked to use a template, select "Static Website".

Reviewer: It doesn't show up like that, it shows up as `static-website`.

Alternatively, you can use the Altostra CLI to initialize a new project from the template by running:
```sh
$ alto init --template static-website
```

You can also apply the template to an existing Altostra project from Visual Studio Code by going 

> to the Altostra view in the main toolbar and clickign on "Static Website" in the templates list.

Reviewer: Make sure it appears that way.

### Required configuration

> This template contains a few parameters that must be 
> [set in the the environment](https://docs.altostra.com/howto/envs/manage-environments.html#manage-configuration-values)
> in which the deployment was created.

Reviewer: ```suggestion
This template contains a few parameters that must be 
[set in the environment](https://docs.altostra.com/howto/envs/manage-environments.html#manage-configuration-values) to which the project will be deployed.
```

> You may also provide all or part of the configuration when 
> [creating or updating the deployment](https://docs.altostra.com/reference/CLI/commands/altostra-cli-deploy.html#providing-configuration).

Reviewer: Let's keep it simple and not give them all the options here in the readme.
```suggestion
```

#### Template parameters

- `DOMAIN`:  
The fully qualified root domain of your application or arganization.
- `DOMAIN_HOST_NAME`:  
A host name for the site CDN.  
It is combined with `DOMAIN` to create a fully qualified domain for the CDN.  
I.E. If we set `DOMAIN: example.com` and `DOMAIN_HOST_NAME: my-site` the CDN would
accept requests for `my-site.example.com`.
- `DOMAIN_CERT`:  

> An Amazon certificate manager (ACM) certificate ARN approprite for the domain.

Reviewer: I think you can remove this after Yossi's change.

### Project deployment

> Here are simple instruction to get you started.  
> For more information on each command please consult [Altostra docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).

Reviewer: ```suggestion
Run the following commands to create a deployment image of the project and deploy it as a new instance.  
For more information on each command refer to the [Altostra CLI docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).
```

> 1. Create an image from the project:

Reviewer: Link the word _Image_ to the docs.

```shell
$ altos push v1.0
```

> 2. Deploy the image to a new deployment named `main` in the `Dev` environment:

Reviewer: You can't just throw the reader at the Dev environment if it doesn't exist. Explain that and/or provide instructions to create one.

Author: I can
It is in the prerequisites list, with instructions

```shell
$ alto deploy main:v1.0 --new Dev
```
3. Update the deployment with new images:
```shell

> $ alto deploy main:v2.0

Reviewer: No need for an update if nothing has changed. You make it look like it's part of the required steps.
A better step would be to run `alto deployments main`.

```
4. Manage the project in the Altostra Web Console:
```shell
$ alto console
```

### Upload and invalidate site content:

> 1. Upload content of all S3 buckets of `main` deployment:

Reviewer: Add a text before this to explain why these steps are necessary, that the content upload is split from the infrastructure deployment.

```shell
$ alto sync main --all --public
```
2. Invalidate all CDN caches of `main` deployment:
```shell
$ alto invalidate main --all
```

## Content

The template is made of three resources: 
- An S3 Bucket
- A CloudFront CDN

> - Domain Name

Reviewer: Update to Yossi's changes.

When asked to use a template, select "static-website".

Alternatively, you can use the Altostra CLI to initialize a new project from the template by running:
```sh
$ alto init --template static-website
```

You can also apply the template to an existing Altostra project from Visual Studio Code by going 
to the Altostra view in the main toolbar and clickign on "static-website" in the templates list.

### Project deployment

Run the following commands to create a deployment image of the project and deploy it as a new instance.

For more information on each command refer to the [Altostra CLI docs](https://docs.altostra.com/reference/CLI/altostra-cli.html).

1. Create an
[image](https://docs.altostra.com/howto/projects/deploy-project.html#create-a-project-image)
from the project:
```shell
$ altos push v1.0
```
2. Deploy the image to a new deployment named `main` in the `Dev` environment:
```shell
$ alto deploy main:v1.0 --new Dev
```
3. Manage the project in the Altostra Web Console:
```shell
$ alto console
```

> To update an existing deployment with new images just omit the `--new` flag and environment name:
> ```shell
> $ alto deploy main:v2.0
>```

> ### Upload and invalidate site content:

Reviewer: ```suggestion
### Upload and invalidate site content
```
